exp-player/addon/components/exp-lookit-geometry-alternation/component.js

```diff
@@ -328,13 +545,19 @@ export default ExpFrameBaseUnsafeComponent.extend(FullScreen, VideoRecord,  {
         $(document).on('keyup.pauser', function(e) {_this.handleSpace(e, _this);});
 
         // Start placeholder video right away
-        _this.sendTimeEvent('exp-alternation:startIntro');
+        /**
+         * Immediately before starting intro/announcement segment
+         *
+         * @event startIntro
+         */
+        _this.sendTimeEvent('startIntro');
         $('#player-video')[0].play();
 
         // Set a timer for the minimum length for the intro/break
         $('#player-audio')[0].play();
         _this.set('introTimer', window.setTimeout(function() {
             _this.set('completedAttn', true);
+            _this.notifyPropertyChange('readyToStartCalibration');
```

Comment:
Hrm, I was aware that it was *supposed* to, but this practice was indeed started because it *didn't* (reproducible difference in behavior between including the notifyPropertyChange call and not). It's possible, though, that that was in a narrower set of circumstances than I now superstitiously make calls to notifyPropertyChange, so I can double-check going forward that this is necessary.